package.json

```diff
@@ -8,8 +8,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "gulp test",
-    "start": "gulp watch"
+    "test": "./node_modules/.bin/mocha test/*",
+    "start": "./node_modules/.bin/mocha test/* --reporter nyan --watch"
```

Comment:
It's not necessary to provide full path in NPM scripts - just `mocha` command here will resolve to `node_modules/.bin/mocha`, anyway.